src/Console/Command/HelpCommand.php

```diff
@@ -122,8 +129,12 @@ protected function asText(ConsoleIo $io, iterable $commands): void
         foreach ($grouped as $prefix => $names) {
             $io->out("<info>{$prefix}</info>:");
             sort($names);
-            foreach ($names as $name) {
-                $io->out(' - ' . $name);
+            foreach ($names as $data) {
+                if ($data['description'] !== '') {
+                    $io->out(' - ' . str_pad($data['name'], 50) . ' | ' . $data['description']);
```

Comment:
Should we pad to the longest name plus some margin? There is a chance we could have a 50+ command name in userland code.

Another approach would be to have the help indented:

```text
cake bake model
    Make models and other interesting things
cake bake controller
    Make controllers and stuff like that.
```

This will avoid any line length related problems but uses more vertical space :shrug:

Reply:
And commands without any description will have an empty line?

Reply:
Yeah or no extra line could work too. We can also apply styling to the description or command name to help with visual separation.